tap_mysql/client.py

```diff
@@ -41,6 +42,48 @@ def patched_conform(
 class MySQLConnector(SQLConnector):
     """Connects to the MySQL SQL source."""
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the SQL connector.
+
+        This method initializes the SQL connector with the provided arguments.
+        It can accept variable-length arguments and keyword arguments to
+        customize the connection settings.
+
+        Args:
+            *args: Variable length argument list.
+            **kwargs: Arbitrary keyword arguments.
+        """
+        super().__init__(*args, **kwargs)
+        # Check if we are using PlanetScale,
+        # if so we need to let our connector know
+        # Ideally we'd just check to see if we're on Vitess,
+        # but I don't know how to do that quickly
+        self.is_vitess = False
+
+        if self.config.get("is_vitess") is None:
```

Comment:
What happens where if `{"is_vitess": true}`?

Reply:
Can't believe I missed that, I just pushed up a fix!